src/formats/onenote.ts

```diff
@@ -664,6 +673,27 @@ export class OneNoteImporter extends FormatImporter {
 		return returnPath;
 	}
 
+	// Helper function to sanitize OCR text for markdown
+	private sanitizeOCRText(text: string): string {
+		// Only keep alphabetic characters and spaces
+		text = text.replace(/[^a-zA-Z\s]/g, '');
+		
+		// Replace multiple spaces with single space
+		text = text.replace(/\s+/g, ' ');
```

Comment:
Trim here before truncating.

```suggestion
		text = text.replace(/\s+/g, ' ').trim();
```

Then return the truncated value.